library/src/main/java/io/karn/notify/entities/Payload.kt

```diff
@@ -167,6 +167,19 @@ sealed class Payload {
             var showTimestamp: Boolean = true
     )
 
+    /**
+     * Contains configuration that is specific to the progress of a notification.
+     */
+    class Progress constructor(
+
+            open var showIndeterminate: Boolean = true,
+
+            open var determinateProgress: Int = 1,
```

Comment:
Since we have an opportunity to change how this is exposed to library users, let's update this to be a little more clear. Something like `progressPercent` should be good, we can set the default to `0` just to keep it consistent.

Reply:
ready